.github/workflows/playwright.yml

```diff
@@ -2,8 +2,6 @@ name: Playwright Tests
 on:
   push:
     branches: [ main ]
-  pull_request:
```

Comment:
Sadly we have to dev environment, so playwright isn't going to be very reliable to automate here.